tests/commons/clean/test_blank_nodes.py

```
@@ -0,0 +1,165 @@
+                "@id": node_id,
+                "@type": "ContactPoint",
+                "email": "mailto:info@example.org",
+                "name": "Test Node"
+            }
+        ]
+    }
+
+    g = Graph()
+    
+    if should_fail_parse:
+        with pytest.raises(Exception):
```

Ruff (0.15.21)

[warning] Do not assert blind exception: `Exception`

(B017)